spec/migrations/migration_spec.rb

```diff
@@ -7,29 +7,68 @@ class MockMigration
 describe ContentfulModel::Migrations::Migration do
   subject { MockMigration.new }
 
-  it '#create_content_type' do
-    expect(ContentfulModel::Migrations::ContentTypeFactory).to receive(:create)
+  before do
+    @mock_ct = @mock_cf = Object.new
```

Comment:
Please define this as 2 separate objects, and change `@mock_cf` to `@mock_field` as it is quite confusing otherwise, it took me several re-reads to understand what it was.

Cheers